.travis.yml

```diff
@@ -7,6 +7,7 @@ services: docker
 
 go:
 - 1.12.12
+- 1.15.7
```

Comment:
I would consider excluding a lot of variants
to save limited CI resources: for analogy please see: 

https://github.com/etcd-io/etcd/blob/7f97dfd45ac82518716a59c246e2c2a95c9825c7/.travis.yml#L37

I think it's sufficient to run:
 - linux-amd64-fmt
 - linux-amd64-unit
 - linux-amd64-integration-4-cpu
 - linux-amd64-functional
 - linux-amd64-grpcproxy
 - all-build

Reply:
Got it, makes sense will do!

This reminds me actually, about the actual version, I wanted to use the latest go 1.15.x which is v1.15.11 but it seems there is no tagged image version for this go version, how does that work?